Package.swift

```diff
@@ -5,7 +5,8 @@ import PackageDescription
 let package = Package(
     name: "nats-swift",
     platforms: [
-        .macOS(.v13)
+        .macOS(.v13),
+        .iOS(.v16)
```

Comment:
v15 won't do?

Reply:
Unfortunately no, there are a 2 things we're using requiring v16 - `Duration` and `ContinuousClock`

Reply:
ah! should we back down? looks like 15 is still supported https://en.wikipedia.org/wiki/IOS_version_history

Reply:
We could, as a follow up PR though.